contracts/core/Eip7702Support.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.8;
+// SPDX-License-Identifier: MIT
+// solhint-disable no-inline-assembly
+
+import "../interfaces/PackedUserOperation.sol";
+import "../core/UserOperationLib.sol";
+
+// EIP-7702 code prefix. Also, we use this prefix as a marker in the initCode. To specify this account is EIP-7702.
+bytes3 constant EIP7702_PREFIX = 0xef0100;
```

Comment:
For the record, I suggest here splitting the two values as there isn't actually an overlap in the meaning:

```
// EIP-7702 code prefix
uint256 constant EIP7702_PREFIX = 0xef0100;
// Prefix as a marker in the "initCode" to specify this account is EIP-7702
uint256 constant EIP7702_PREFIX = 0xff7702;
```